BurnOutSharp/ProtectionType/AACS.cs

```diff
@@ -12,23 +12,49 @@ public string CheckPath(string path, bool isDirectory, IEnumerable<string> files
         {
             if (isDirectory)
             {
-                if (files.Any(f => f.Contains(Path.Combine("aacs", "VTKF000.AACS")))
-                    && files.Any(f => f.Contains(Path.Combine("AACS", "CPSUnit00001.cci"))))
+                if (files.Any(f => f.Contains(Path.Combine("AACS", "MKBROM.AACS"))))
                 {
-                    return "AACS";
+                    string versionPathHDDVD = files.FirstOrDefault(f => Path.GetFileName(f).Equals("MKBROM.AACS", StringComparison.OrdinalIgnoreCase));
+                    int? version = GetVersionHDDVD(versionPathHDDVD);
+                    return $"AACS Version {version}";
```

Comment:
You shouldn't be returning this string if `version` is `null`. Something like `AACS (Unknown Version)` would be better.